src/control/cmd/dmg/system.go

```diff
@@ -207,14 +207,14 @@ func displaySystemAction(log logging.Logger, results system.MemberResults) error
 type systemStopCmd struct {
 	logCmd
 	connectedCmd
-	Force bool `long:"force" description:"Force stop DAOS system members"`
```

Comment:
Maybe I misunderstood in the call - I thought we were leaving the option in, to keep the interface the same. It just wouldn't do anything right now.

Reply:
Yes, correct. We should leave the option in and just make it the default behind the scenes for this release so that the user doesn't need to supply it.

Reply:
what's the point of leaving the option in if it is completely ineffectual?

Reply:
surely that's misleading and someone is going to expect modified behaviour if supplying an option